### ISSUE TITLE
Release new version of the sqlalchemy-exasol package on pypi

### ISSUE THREAD

**Issue by Nicoretti** [exasol/sqlalchemy-exasol#123]
Since the last release of [sqlalchemy-exasol](https://pypi.org/project/sqlalchemy-exasol/) (2.2.0) in september 2020, a couple of small changes and bugfixes have been taken place within the project, therefore a new release needs to be crated.

:warning: Due to the fact that this is the first release, since exasols take over of the project, a couple of task(s) need to be done beforehand. :warning: 

## Task(s)
- [x] Update `setup.py`
   - [x] Update mail contact `opensource@exasol.com`
   - [x] Update authors
   - [x] Update outdated dependency markers
   - [x]  Update project url
- [x] Update README
   - [x] Remove "Get EXASolution database" and replace with Exasol-DB  as requirement
- [x] Add link checker
   - [x] Fix outdated links
- [x] Make sure PyPi ~~and TestPyPi~~ account~~s are~~ is setup correctly
   (Note: PyPi Test have been postponed, clarification on exasol account needed)
   - [x] Accounts
   - [x] API Token
- [x] Make sure secrets are up to date
- [x] understand versioneer + existing release process
- [x] Update changelog

## Release
- [x] Tag new version
- [x] Release
